lib/main.dart

```diff
@@ -8,6 +8,8 @@ import 'package:sentry_mobile/redux/actions.dart';
 import 'package:sentry_mobile/redux/middlewares.dart';
 import 'package:sentry_mobile/redux/reducers.dart';
 import 'package:sentry_mobile/redux/state/app_state.dart';
+import 'package:sentry_mobile/screens/release_health/release_health.dart';
+import 'package:sentry_mobile/settings.dart';
```

Comment:
what's about relative imports? that should be the way to go for our own classes

https://github.com/erluxman/awesomefluttertips#tip-10--relative-import--the-right-way-to-import-dart-files-we-have-in-our-lib-package

Reply:
Hmm, not sure if this is the best way now that i've changed it. Imports are always relative to the file wehre we import, so for example we have imports like `import '../../api/api_errors.dart';` if we are in a folder, which takes some time to write. Since i already made the change project wide (i know), lets stick with it and see if we feel more comfortable with it.

Reply:
sounds good, I usually let the IDE importing for me :)